daemon/mgr/cri.go

```diff
@@ -290,7 +290,18 @@ func (c *CriManager) ListImages(ctx context.Context, r *runtime.ListImagesReques
 
 // ImageStatus returns the status of the image, returns nil if the image isn't present.
```

Comment:
What does the status of image mean? The existence of the image, or the ImageInfo as meta data of image?

Reply:
`ImageStatus` will return the **Basic information** about a container image includes "ID" and "Size" and **extra information about the image** if needed. Also, if the image isn't present,it will return nil.

Reply:
Updated `4.Describe how to verify it`

Reply:
It is some kind of weird to use status, Maybe info is better. Not blocking this pr.